src/app/containers/Ad/Amp/index.jsx

```diff
@@ -18,6 +18,11 @@ import AdSlot from './AdSlot';
 
 const FullWidthWrapper = styled.div`
   background-color: ${C_LUNAR_LIGHT};
+
+  .amp-geo-group-gbOrUnknown & {
```

Comment:
Where are the geo groups specified and where is this class added by AMP?

Reply:
These are added by https://github.com/bbc/psammead/blob/latest/packages/components/psammead-amp-geo/src/index.jsx. `amp-geo` adds and removes classes to the body of the document: https://amp.dev/documentation/components/amp-geo - I have linked to the relevant section elsewhere, but maybe it's worth explaining in a comment here too. I think it's quite difficult to understand this behaviour unless you've read up on how the component works, so I agree it could use some explanation.

Reply:
Yeah, I saw you linked it in https://github.com/bbc/simorgh/pull/6961/files#diff-3e335d5559a68f6c5fd831d451b1ce70R44, but it would great to also explain here how AMP will remove the `amp-geo-pending` class and add the `amp-geo-group-` classes to hide the ads for those groups.